Library/Formula/rack.rb

```diff
@@ -0,0 +1,27 @@
+class Rack < Formula
+  desc "CLI for Rackspace"
+  homepage "https://github.com/rackspace/rack"
+  url "https://github.com/rackspace/rack.git",
+      :tag => "1.0.1",
+      :revision => "71a8d7c80b3652b4dd39e683bc423b8a542b0167"
+  head "https://github.com/rackspace/rack.git"
+
+  depends_on "go" => :build
+
+  def install
+    ENV["GOPATH"] = buildpath
+    ENV["TRAVIS_TAG"] = @tag
+
+    rackpath = buildpath/"src/github.com/rackspace/rack"
+    rackpath.install (Dir["{*,.*}"] - [".", ".."])
+
+    cd rackpath do
+      system "script/build", "rack"
+      bin.install "rack"
+    end
+  end
+
+  test do
+    system "#{bin}/rack"
```

Comment:
I'd love to, but I'm not sure we can. This is a tool for accessing a public API; is doing that considered acceptable? (Even if it is, I'm not sure said API does anything useful before asking for authentication; although I suppose checking that you can even authenticate is already better than `--version`.)